lib/commands/analyze-dependency.ts

```diff
@@ -38,13 +41,18 @@ export const builder: yargs.CommandBuilder = {
 
 export async function handler(argv: yargs.Arguments): Promise<void> {
   await cliSuppressExceptions(async () => {
-    const readmeMd: string = argv.readme;
-
-    const autorestConfig = await getAutorestConfig(argv, readmeMd);
-    const fileRoot = path.dirname(readmeMd);
-    const swaggerFilePaths: string[] = autorestConfig["input-file"].map((it: string) =>
-      path.resolve(fileRoot, it)
-    );
+    let swaggerFilePaths: string[] = [];
+    if (argv.readme !== undefined) {
+      const readmeMd: string = argv.readme;
+      const autorestConfig = await getAutorestConfig(argv, readmeMd);
+      const fileRoot = path.dirname(readmeMd);
+      swaggerFilePaths = autorestConfig["input-file"].map((it: string) =>
+        path.resolve(fileRoot, it)
+      );
+    }
+    if (argv.swagger !== undefined) {
```

Comment:
Seems swagger input will take priority when both swagger and readme are specified. Can you add this information in the command line help output?

Reply:
Good suggestion.